boilerplate/package.json.ejs

```diff
@@ -61,7 +61,14 @@
     "typescript": "2.8.3"
   },
   "jest": {
-    "preset": "jest-preset-ignite"
+    "preset": "jest-preset-ignite",
+    "testPathIgnorePatterns": [
+      "/e2e/",
```

Comment:
This line is only necessary if using Detox (Bowser does not) and the Detox tests are located in the `e2e` folder, which I'm not certain is the default, so I would probably suggest removing this line.

Reply:
yeah, will remove.

Reply:
@kevinvangelder removed this line in package.json. Final verion

```
  "jest": {
    "preset": "jest-preset-ignite",
    "testPathIgnorePatterns": [
      "/node_modules"
    ],
    "globals": {
      "__TEST__": true
    }
  },
```